### PR TITLE
fix: remove moderation from how to

### PR DESCRIPTION
**Pull request by thisislawatts**
PR Checklist

- [X] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

- chore: remove accept/deny buttons from how-to description
- fix: load how-to moderation comments from new property
- fix: remove unused moderation methods

Address https://github.com/ONEARMY/community-platform/issues/2876

Also restyles feedback to appear within standard Alert component using the info variant:

Before:

Screenshot 2023-10-21 at 18 27 00

After:

Screenshot 2023-10-21 at 18 25 58

### CODE REVIEW

**Review by evakill — COMMENTED** [review]
this looks good to me - not sure if was intentionally left as draft

**Comment by thisislawatts** [comment]
Ah sorry @evakill, thanks for taking a look. I have a habit of only switching PRs out of Draft once all the tests have passed :)

File: src/models/howto.models.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ export interface IHowto extends IHowtoFormInput {
   total_downloads?: number
   mentions: UserMention[]
   previousSlugs: string[]
+  moderatorFeedback?: string
```
Comment (evakill) [inline src/models/howto.models.tsx:26]: since `IHowtoFormInput` extends `IModerable` we don't need to add this here

**Comment by evakill** [comment]
otherwise LGTM

File: src/models/howto.models.tsx (hunk 1 above)
Comment (thisislawatts) [reply src/models/howto.models.tsx]: Fixed :) Thanks for catching that.

**Review by evakill — APPROVED** [review]

**Pull request merged**